### DISCUSSION TITLE
What are the three dots in current line?

### DISCUSSION THREAD

**Discussion by renhiyama** [ajaxorg/ace#5024] [Q&A] [answered]
![Screenshot_20230117_123844_Kiwi Browser](https://user-images.githubusercontent.com/92968534/212832519-df293ae7-c0d6-471c-b6e1-383452b218ff.jpg)

What is this three dots button and how can I get rid of it? I tried unticking all the options in kitchen sink but it is still there :(

**Comment by nightwing** [comment, marked as answer]
It is actions menu for mobile devices, easiest way to hide it is to use css 

```css
.ace_mobile-menu {
    display: none;
}
```

**Reply by renhiyama** [reply]
Thanks!
